tests/unit/help_test.sh

```diff
@@ -32,4 +32,74 @@ function test_kworkflow_man()
   assertEquals "($LINENO) We expected an error message." "$expect" "$output"
 }
 
+function test_kworkflow_version()
+{
+  local KW_LIB_DIR
+  local output
+  local expected_output
+
+  # the mocked version for KW
+  expected_output=$(printf 'beta\nBranch: bazz\nCommit: ffddee\n')
+
+  # this value mocks a fake location for the version file, which is the file used
+  # by KW to get information about the current version.
+  KW_LIB_DIR="${SHUNIT_TMPDIR}"
+  printf '%s' "${expected_output}" > "${KW_LIB_DIR}/VERSION"
+
+  # check output
+  output=$(kworkflow_version)
+  assertEquals "(${LINENO}) Got wrong kw version." "${expected_output}" "${output}"
```

Comment:
Prefer the use of `assert_equals_helper` over `assertEquals` when comparing complete strings, as the failing output is much better for the latter. As an illustration, the first screenshot below is the output of this assertion using `assertEquals`, and the second is the output using `assert_equals_helper`.

![Screenshot from 2023-12-23 17-31-31](https://github.com/kworkflow/kworkflow/assets/95055784/f34edc41-60df-453e-929a-a531dd8a62cd)
![Screenshot from 2023-12-23 17-32-26](https://github.com/kworkflow/kworkflow/assets/95055784/c0c728b4-1ec2-43d1-afa9-d09028f9c1d5)